traffic_ops/bin/traffic_ops_ort.pl

```diff
@@ -1623,9 +1623,7 @@ sub check_lwp_response_content_length {
 	my $url           = $lwp_response->request->uri;
 
 	if ( !defined($lwp_response->header('Content-Length')) ) {
-		( $log_level >> $panic_level ) && print $log_level_str . " $url did not return a Content-Length header!\n";
-		exit;
-		return 1;
+		return 0; # Content-Length MAY be omitted per HTTP/1.1 RFC 7230, and in fact MUST NOT be included with a 'Transfer-Encoding: Chunked' header, which MUST be accepted by clients.
```

Comment:
At one point in time, Traffic Ops was happily serving incomplete files (potential myriad of errors causing this), and ORT was happily writing them to disk, and telling ATS to apply this partial config. Most of the time, ATS would reject this partial config, as it was invalid, and keep running with the old config. At some point later, ATS would crash and be forced to load the config from disk, and would not start with this invalid config. So, this check was crucial in the stability of our overall CDN.